website/common/locales/en/front.json

```diff
@@ -253,10 +253,11 @@
     "usernameTaken": "Username already taken.",
     "passwordConfirmationMatch": "Password confirmation doesn't match password.",
     "invalidLoginCredentials": "Incorrect username and/or email and/or password.",
+    "passwordResetPage": "Reset Password",
     "passwordReset": "If we have your email on file, a new password has been sent to your email.",
     "passwordResetEmailSubject": "Password Reset for Habitica",
-    "passwordResetEmailText": "Password for <%= username %> has been reset to <%= newPassword %> . Important! Both username and password are case-sensitive -- you must enter both exactly as shown here. We recommend copying and pasting both instead of typing them. Log in at <%= baseUrl %>. After you have logged in, head to <%= baseUrl %>/#/options/settings/settings and change your password.",
-    "passwordResetEmailHtml": "Password for <strong><%= username %></strong> has been reset to <strong><%= newPassword %></strong><br /><br />Important! Both username and password are case-sensitive -- you must enter both exactly as shown here. We recommend copying and pasting both instead of typing them.<br /><br />Log in at <%= baseUrl %>. After you have logged in, head to <%= baseUrl %>/#/options/settings/settings and change your password.",
+    "passwordResetEmailText": "If you requested a password reset for <%= username %> on Habitica head to <%= passwordResetLink %> to set a new one. The link will expire after 24 hours. If you haven't request a password reset please ignore this email.",
```

Comment:
It's good! Just "request" in the final sentence should be "requested" and I'd put a comma after "Habitica".